test/IRGen/objc_class_in_storage_decls.swift

```diff
@@ -0,0 +1,49 @@
+// RUN: %target-swift-frontend -emit-ir -disable-objc-attr-requires-foundation-module %s | %FileCheck %s
+
+struct WrapperStruct {
+  subscript() -> Void {
+    unsafeAddress {
+      // CHECK: @"OBJC_CLASS_$__TtCFV27objc_class_in_storage_decls13WrapperStructlu9subscriptFT_T_L_31ClassInSubscriptUnsafeAddressor"
```

Comment:
New tests look good! *Technically* the thing we care about is the string in the class metadata…

```
@5 = private unnamed_addr constant [79 x i8] c"_TtCFV4main13WrapperStructlu9subscriptFT_T_L_31ClassInSubscriptUnsafeAddressor\00"
```

…but this is probably close enough. Thanks, Alex!

(letting Erik handle the actual review)